metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -748,7 +748,6 @@ def _generate_single_profile(
         custom_sql: str = None,
         **kwargs: Any,
     ) -> Optional[DatasetProfileClass]:
-        logger.info(f"Profiling {pretty_name}")
```

Comment:
intentional?

Reply:
yep, I left here earlier non intential and now we duplicate this message